server/config.py

```diff
@@ -9,6 +9,5 @@
 SERVICE_ROOT = 'https://cida.usgs.gov'
 STATS_SERVICE_ROOT = 'http://cida.usgs.gov/ngwmn_statistics'
 CONFLUENCE_URL = 'place holder - see dev tier for workspace value'
-STATISTICS_METHODS_URL = 'place holder - see dev tier for workspace value'
```

Comment:
Should the related view be removed as well? https://github.com/ACWI-SOGW/ngwmn-ui/blob/761d101541868e509a4a381ddc24ff53581daf93/server/ngwmn/views.py#L49

Reply:
not exactly. First, Eric did not find that page in the export he used to generate the content service. Second, I used pycharm to search for uses of the constant and it found none. I was concerned that there might be a use that pycharm could not find. Thanks Mary, we need to discuss this with Candice for a course of action. I would like to keep this open until we know if we need to cancel it or also remove the view.